### DISCUSSION TITLE
Looking for coding help

### DISCUSSION THREAD

**Discussion by everywherebrand** [community/community#112881] [New to GitHub]
Hi there - new to GiHub but through it would be a good place to get some help to see if an idea I have is viable and how to go about it? 
Basically is something that can scrape multiple websites for basic things like colours used. maybe all colours used within the logo too.
Thanks all, David

**Comment by ghost** [comment]
Hi @everywherebrand if you are planning to work on a web-scraping project ,  **python** can come in handy 
you can use **Beautifulsoup** library for this matter , start with reading some docs or maybe tutorials.

your idea seems unique but i have a suggestion if it's all about gathering colours ... it could be a palette maker too ! 
for instance : css themes recommendation tool ...
Good luck.

**Reply by everywherebrand** [reply]
Thanks @KohoolateSen - yes, beautiful soup has come up in some chats I've had - I"m not sure of it's limitations and if it could get colour info from items like the main logo - but maybe developers of all sites would include the colour in the logo svg/png in the pallet?
Guess I just need to start experimenting - but I'm def not a coder (more creative) so I'm looking for someone to start experimenting with & see the scale of the challenge I may have!
Is it something you could look in to?
Thank you & عشق و آرامش برای همه در ایران

**Reply by ghost** [reply]
Thanks for your kindness
Yeah sure , Basically an image is a matrix of colored pixels so when you process an image let's say with **pillow** library in python , all it does is it gives you a list of colors  which are used to make up the image.

But you'll also need **beautifulsoup** to scrape websites and grab those logo/images

With just a bit being smart you can detect colors which are mostly used in the logo
as main colors ( for the sake of avoiding confusion and outlier data )

I'll tell you to give it a shot 
Being a coder also requires to be creative :)

**Reply by everywherebrand** [reply]
Thank you! That sounds optimistic! 
Yes - def agree that coding=creativity but I'm afraid my creativity has limits!!
Is GitHub a good place to look to commission coders (obvs ££)

**Reply by ghost** [reply]
Don't be scared you're good , 
Umm I don't know exactly 
But GitHub is mainly a platform designed to preserve FOSS ( free and open-source software ) conventions I mean  if I got your point precisely, you can lookup freelancing platforms like Fiverr to look to commission coders.

**Reply by everywherebrand** [reply]
Prefect - you've def helped me understand the kind of approach I'd be looking for as I look for freelancers! I'm David @ Everywherebrand - if there's any help you need on a visual creative side and or help in the UK then def get in contact!

**Reply by ghost** [reply]
Anytime ! ofc :)
I'm Sadra or @KohoolateSen 
Feel free to ask any question that I might know about. I'll follow u to have your account.

Before I forget please mark this discussion as answered if it was helpful. 

Good luck & Have fun.

**Comment by jfullstackdev** [comment]
you can start your project and post questions in here,

**Comment by Anzalone394** [comment]
hello

I reached out to you in order to have a better chat after seeing your article.
Are you still in need of assistance? If so, please let me know so that we may have a more in-depth conversation.
message me on discord: bushraa03
